README.md

```diff
@@ -1,59 +1,24 @@
+[![Push Events](https://github.com/agrc/deq-eid-skid/actions/workflows/push.yml/badge.svg)](https://github.com/agrc/skid/actions/workflows/push.yml)
+[![Pull Events](https://github.com/agrc/deq-eid-skid/actions/workflows/pull_request.yml/badge.svg)](https://github.com/agrc/skid/actions/workflows/pull_request.yml)
 
-A template for building skids that use palletjack to update AGOL data from a tabular data source and that are run as Google Cloud Functions
+This skid pulls data from the DEQ EID Salesforce instance and loads it into DEQ AGOL hosted feature layers [for use in the interactive map](https://github.com/agrc/deq-enviro/issues/665).
 
-For an example of a working skid, see [erap-skid](https://github.com/agrc/erap-skid) or [uorg-skid](https://github.com/agrc/uorg-skid).
-
-## Creating a Git Repository
-
-The first step in creating a skid is to create a new repository in GitHub that uses this repo as a template. You'll then clone the repo to your computer and start developing.
-
-1. Create a new repo in <https://github.com/agrc>
-   - Under `Repository template`, choose `agrc/skid`
-   - Name it `projectname-skid` so that everyone knows it's a skid
-   - Make it a Public repo
-   - Leave everything else alone (the template will take care of it all).
-1. Clone the repo on your local computer
-   - Use GitHub Desktop or any terminal with the git cli installed.
-   - git cli commands:
-      - `cd c:\where\you\store\git\repos`
-      - `git clone https://github.com/agrc/projectname-skid`
-
-## Initial Skid Development
-
-You'll need to do a few steps to set up your environment to develop a skid. You may also want to make sure you've got the skid working locally before adding the complexity of the cloud function.
-
-This all presumes you're working in Visual Studio Code.
+## Development Setup
 
 1. Create new environment for the project and install Python
-   - `conda create --name PROJECT_NAME python=3.8`
-   - `conda activate PROJECT_NAME`
-1. Open the repo folder in VS Code
-1. Rename `src/skidname` folder to your desired skid name
-1. Edit the `setup.py:name, url, description, keywords, and entry_points` to reflect your new skid name
-1. Edit the `test_skidname.py` to match your skid name.
-   - You will have one `test_filename.py` file for each program file in your `src` directory and you will write tests for the specific file in the `test_filename.py` file
+   - `conda create --name deq-eid-skid python=3.11`
+   - `conda activate deq-eid-skid`
 1. Install the skid in your conda environment as an editable package for development
    - This will install all the normal and development dependencies (palletjack, supervisor, etc)
    - `cd c:\path\to\repo`
    - `pip install -e .[tests]`
-   - add any additional project requirements to the `setup.py:install_requires` list
 1. Set config variables and secrets
    - `secrets.json` holds passwords, secret keys, etc, and will not (and should not) be tracked in git
```

Comment:
**suggestion**: for the skids that run in the cloud we should start migrating to secret manager

Reply:
This is using secret manager. It sets a single secret as a JSON value. Or am I missing something?